src/quicktype-core/language/CPlusPlus.ts

```diff
@@ -400,88 +499,166 @@ export class CPlusPlusRenderer extends ConvenienceRenderer {
     }
 
     protected emitClassMembers(c: ClassType): void {
-        this.forEachClassProperty(c, "none", (name, jsonName, property) => {
+        if (this._options.withGetterSetter) {
+            this.emitLine("private:");
+            this.ensureBlankLine();
+
+            this.forEachClassProperty(c, "none", (name, _jsonName, property) => {
+                this.emitLine(this.cppType(property.type, { needsForwardIndirection: true, needsOptionalIndirection: true, inJsonNamespace: false }, true), " ", name, ";");
+            });
+
+            this.emitLine("public:");
+            this.ensureBlankLine();
+        }
+
+        this.forEachClassProperty(c, "leading-and-interposing", (name, jsonName, property) => {
             this.emitDescription(this.descriptionForClassProperty(c, jsonName));
-            this.emitLine(
-                this.cppType(
-                    property.type,
-                    { needsForwardIndirection: true, needsOptionalIndirection: true, inJsonNamespace: false },
-                    true
-                ),
-                " ",
-                name,
-                ";"
-            );
+            if (!this._options.withGetterSetter) {
+                this.emitLine(this.cppType(property.type, { needsForwardIndirection: true, needsOptionalIndirection: true, inJsonNamespace: false }, true), " ", name, ";");
+            } else {
+                const [getterName, mutableGetterName, setterName] = defined(this._gettersAndSettersForPropertyName.get(name));
+                const rendered = this.cppType(property.type, { needsForwardIndirection: true, needsOptionalIndirection: true, inJsonNamespace: false }, true);
+                this.emitLine("const ", rendered, " & ", getterName, "() const { return ", name, "; }");
+                this.emitLine(rendered, " & ", mutableGetterName, "() { return ", name, "; }");
+                this.emitLine("void ", setterName, "( const ", rendered, "& value) { ", name, " = value; }");   
```

Comment:
I think our C++ styles are incompatible.  The rest of the renderer emits `void func(int x)` but you do `void func( int x )`.  I'm happy with either - again, I defer to your judgement - but it should be consistent.

Reply:
Oopsie, extra space removed.